### DISCUSSION TITLE
Analytics expensive?

### DISCUSSION THREAD

**Discussion by PrimozRome** [vercel/community#2577] [Idea]
Anyone else sees Vercel analytics to be a little bit expensive?

We have about 100k visitors / month and I will have to upgrade to Vercel Analytics PRO for 50$/month as the 25 000 events is overpassed in couple of days. I would still do it, but then I see this line... that's ridiculous.

<img width="493" alt="Screenshot 2023-05-24 at 13 37 32" src="https://github.com/vercel/community/assets/1180467/804bdd24-0e8d-432f-b91d-d95b7d07b7b1">

Splitbee didn't have these limitations before it was purchased by Vercel... And was available for 10$/month. And it included some useful additional features like Funnels and A/B testing. Now 50$ really seems expensive and something that will steer me, and I am sure somebody else too, back to Google Analytics.

What's your opinion on this?

**Comment by timolins** [comment]
Hey @PrimozRome, thanks for your feedback.

To clarify, if you have less than 500k events you can comfortable stay on the Pro plan. If you go over the 25k events included in the Pro plan, you'll be charged an additional $14 for every 100k events.

As for features, be assured there is a lot more coming. 👍

**Comment by mattiastofte** [comment]
I agree, we'll switch over to something like plausible for this reason. $14 dollars for every 100k events is ridiculous.

**Comment by cflattum** [comment]
Can we disable an automatic charge if we go over the event threshold? I want to see the analytics, but don't care to see them if I get charged for them -
